bookkeeper-server/src/test/java/org/apache/bookkeeper/util/collections/ArrayUtils2Test.java

```diff
@@ -0,0 +1,156 @@
+/*
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+package org.apache.bookkeeper.util.collections;
+
+import java.util.Arrays;
```

Comment:
Nit: reorder imports?

Reply:
Updated, though I'm not sure what order you want (I don't use an IDE).

Reply:
@ivankelly  for me is OK, I think that we have no clear rules about import order, maybe we should discuss on the list. Maybe checkstyle will impose some ordering

//cc @sijie

Reply:
Ya, a checkstyle would be good. However turning it on would result in a bunch of errors straight away. Personally, I have little preference for imports, as long as they look neat.

Reply:
there is already a checkstyle rule. it applies on the modules that have checkstyle enabled. the packages in bookkeeper-server are not enforced yet.

Import Static and then Import. And sort the imports alphabetically. try to avoid start import.

```
        <module name="ImportOrder">
            <property name="severity" value="error"/>
            <!-- This ensures that static imports go first. -->
            <property name="option" value="top"/>
            <property name="sortStaticImportsAlphabetically" value="true"/>
            <property name="tokens" value="STATIC_IMPORT, IMPORT"/>
            <message key="import.ordering"
                     value="Import {0} appears after other imports that it should precede"/>
        </module>

        <module name="AvoidStarImport">
            <property name="severity" value="error"/>
        </module>
```